www/nginx/src/opnsense/scripts/nginx/ngx_autoblock.php

```diff
@@ -0,0 +1,145 @@
+#!/usr/local/bin/php
```

Comment:
does this have to be prefixed "ngx_" ?

Reply:
no, it does not. Is it a problem? It is just the file name of my cron job which must run as often as possible which means every minute.

Reply:
![screenshot_20180926_180727](https://user-images.githubusercontent.com/3713914/46093220-07de8680-c1b7-11e8-96a0-70d139c89a03.png)
Wouldn't it be better to rename the fourth one?

the first one is for header handling, the second and the third one nginx related and the fifth one is the standard setup routine which is called setup in any plugin.

Reply:
it depends on whether you want csp_ ngx_ and so on to have a special meaning for different call context? I was only suggesting name simplification because "nginx" is implied by the parent directory, but I don't know enough about inner workings of nginx to know if "ngx" is something inside the process or something that you want explicitly separated for a specific reason. :)

Reply:
csp_report: API endpoint for CSP violation reports
ngx_auth: API endpoint for authentication request (checkbox advanced authentication)
ngx_autoblock: cron job which looks for bad IPs and sends them to PF
read_log: configd call to read the logs for the frontend (logs view)
setup: create files, chmod, export certs

Reply:
the first one is generic, the 2-3 depend on how nginx or the plugin is working, 4 is generic (would probably also work for other servers), 5 is plugin code

Reply:
ok, thanks. to the original point read_log is perfectly clear so no need to change that